runtime/io.js

```diff
@@ -366,6 +366,7 @@ function caml_ml_flush (chanid) {
 //Requires: caml_ml_flush,caml_ml_bytes_length
 //Requires: caml_create_bytes, caml_blit_bytes, caml_raise_sys_error, caml_ml_channels, caml_string_of_bytes
 //Requires: caml_jsbytes_of_string
+var MAX_LENGTH = 65536;
```

Comment:
where should I place this? At the top of the file and rename it to `MAX_BUFFER_LENGTH`?

Reply:
It's only going to be used here. Just put it inside the funciton.